examples/k8s/k8s-helloworld-default-v1.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 IBM Corporation
```

Comment:
and please add a note about customizing the tests for kubernetes, in  testing/test-scripts/helloworld.sh above the following code snippet.
```bash
 echo "Set/verify the Hello World default route to v1"
 echo "  Invoking a8ctl to set the default route"
 $A8CLI rule-create -f $SCRIPTDIR/helloworld-default-route-rules.json > /tmp/a8ctl_output.txt
```